app/models/case_log.rb

```diff
@@ -289,6 +290,16 @@ def self.to_csv
     end
   end
 
+  def soft_min_for_period
```

Comment:
This might be worth monitoring. Potentially adds a lot of DB lookups although I think rails is smart enough to read from cache...

Reply:
Oooh, that is very true! We call routed_to so often 🤔